fbgemm_gpu/README.md

```diff
@@ -10,14 +10,14 @@ high-performance CUDA GPU operator library for GPU training and inference.
 The library provides efficient table batched embedding bag,
 data layout transformation, and quantization supports.
 
-Currently tested with CUDA 11.3, 11.5, and 11.6 in CI. In all cases, we test with PyTorch packages which are built with CUDA 11.3.
+Currently tested with CUDA 11.3, 11.5, 11.6, and 11.7 in CI. In all cases, we test with PyTorch packages which are built with CUDA 11.3.
```

Comment:
Should it be changed to "In all cases, we test with PyTorch packages which are built with CUDA 11.7" ?

Reply:
Thanks. Updated it to `11.7`.